lesson_28/test_data.py

```diff
@@ -0,0 +1,28 @@
+class TestData:
+
+    _BASE_URL: str = "https://qauto2.forstudy.space"
+
+    _USER_NAME: str = "Mifodii"
+    _USER_LAST_NAME: str = "Aprofiev"
+    _USER_EMAIL: str = "mif_ap_check@gmail.com"
+    _USER_PASSWORD: str = "nihaIdrYT67!."
+
+    @property
```

Comment:
For Test Data is no need to create a properties as you do not create object attributes, just constant variables. It is enough to import var and use it without getter method